src/node/markdownToVue.ts

```diff
@@ -169,9 +169,9 @@ const defaultExportRE = /((?:^|\n|;)\s*)export(\s*)default/
 const namedDefaultExportRE = /((?:^|\n|;)\s*)export(.+)as(\s*)default/
 
 function genPageDataCode(tags: string[], data: PageData) {
-  const code = `\nexport const __pageData = ${JSON.stringify(
+  const code = `\nexport const __pageData = JSON.parse(${JSON.stringify(
```

Comment:
Can't this be simplified to:

```ts
  const code = `\nexport const __pageData = ${JSON.stringify(data)}`
```

Reply:
JSON parsing is faster than object literals, and that's way we are using `JSON.stringify(JSON.stringify())` before 😆

Reply:
FYI: https://github.com/GoogleChromeLabs/json-parse-benchmark